src/ui/views/Player/RatingsOverview.basketball.tsx

```diff
@@ -1,67 +1,175 @@
 import PropTypes from "prop-types";
 import React from "react";
 import type { PlayerRatings } from "../../../common/types.basketball";
+import RatingWithChange from "../../components/RatingWithChange";
 
 const RatingsOverview = ({ ratings }: { ratings: PlayerRatings[] }) => {
 	const r = ratings.length - 1;
+	const lastSeason: PlayerRatings[] =
```

Comment:
I think it would be cleaner if you did something like:

    let lastSeason: PlayerRatings = ratings[r];
    // Search backwards to find the last entry from last season, in the case where there are multiple rows due to injuries
    for (let i = ratings.length - 1; i--; i >= 0) {
        if (ratings[i].season === ratings[r].season - 1) {
            lastSeason = ratings[i];
            break;
        }
    }

and then you can get rid of all the Math.max calls below.

Reply:
(Also the Math.max calls are not even what we want, we'd want Math.min because ratings can currently only go down within a season, and we want to compare with the last ratings from last season. However my suggested change is more future proof, in case eventually there is some way for ratings to increase.)